spec/features/editorial_version_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe 'editorial version details', type: :feature do
+  Warden.test_mode!
+
+  let!(:root_node) { Fabricate(:root_node) }
```

Comment:
The `:root_node` fabricator should not be required. It should be created by the section fabricator.